index.html

```diff
@@ -1,276 +1,443 @@
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                class="social-media-icons"
+                src="./images/icons/media.png"
+                alt="Media Icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                class="social-media-icons"
+                src="./images/icons/github.png"
+                alt="Github Icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                class="social-media-icons"
+                src="./images/icons/victory.png"
+                alt="Victory Icon"
+              />
+            </a>
+          </li>
+        </ul>
+        <div class="btn-container">
+          <button type="submit" class="view-btn">Get my resume</button>
+        </div>
+      </section>
+      <section class="skills-section">
+        <div class="skill-label-container">
+          <h3 class="label">languages</h3>
+          <img src="./images/icons/arrow-down.svg" alt="arrow-down-icon" />
+        </div>
+        <ul class="skill-list">
+          <li>
+            <img src="./images/about-image/Ellipse-html.svg" alt="html-logo" />
+            <span>HTML</span>
+          </li>
+          <li>
+            <img src="./images/about-image/Ellipse-css.svg" alt="css-logo" />
+            <span>CSS</span>
+          </li>
+          <li>
+            <img
+              src="./images/about-image/Ellipse-js.svg"
+              alt="javascript-logo"
+            />
+            <span>JavaScript</span>
+          </li>
+        </ul>
+        <div class="skill-label-container">
+          <p class="label">frameworks</p>
+          <img src="./images/icons/arrow-right.svg" alt="arrowright-icon" />
+        </div>
+        <ul class="skill-list"></ul>
+        <div class="skill-label-container">
+          <p class="label">skills</p>
+          <img src="./images/icons/arrow-right.svg" alt="arrowright-icon" />
+        </div>
+        <ul class="skill-list"></ul>
+      </section>
+      <!--contact section-->
+      <section id="reach-out">
+        <h2 class="contact-me" id="contact">Contact me</h2>
+        <p class="cont-input">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mbjzdeql" method="post">
+          <ul class="ul-form">
             <li>
-               <a href="#">
-                  <img class="social-media-icons" src="./images/icons/linkedin.png" alt="LinkedIn Icon">
-               </a>
+              <input
+                class="username"
+                type="text"
+                name="username"
+                placeholder="username"
+                required
+                maxlength="30"
```

Comment:
- [ ] Great job 👍 . Kindly make sure your `input` tag isn't wrapped in a `ul` tag. 🙏 . It is best practice if they are just wrapped in a `form` tag

Reply:
thank you for the feedback , it's now corrected